libraries/chain/webassembly/runtimes/eos-vm.cpp

```diff
@@ -131,17 +142,29 @@ class eos_vm_instantiated_module : public wasm_instantiated_module_interface {
          _instantiated_module(std::move(mod)) {}
 
       void do_sync_call(apply_context& context) override {
+         const std::optional<sync_call_context>& sync_call_ctx  = context.get_sync_call_ctx();
+         assert(sync_call_ctx.has_value());
+         assert(!context.get_action_ptr());
+
+         uint32_t sync_call_idx = _instantiated_module->get_module().get_exported_function("sync_call");
+         if (sync_call_idx == std::numeric_limits<uint32_t>::max()) {
+            if (sync_call_ctx->no_op_if_receiver_not_support_sync_call()) {
+               dlog("receiver does not have sync call entry point or the entry point is invalid, and the no-op flag is set. just returns");
```

Comment:
Probably do not want to leave in this debug log.

Reply:
https://github.com/AntelopeIO/spring/pull/1266/commits/d76df03a9d578285f79a4cc893dd4a615fdf911c